api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -108,39 +123,46 @@ async function internalGetDocuments({
     throw err;
   }
 
-  const documents: Document[] = docs.entry
-    ? docs.entry.flatMap(d => {
-        if (d.content.size === 0) {
-          capture.message("Document is of size 0", {
-            extra: d.content,
-          });
+  if (cwErrs.length > 0) {
+    capture.message("Document query contained errors", {
+      extra: {
+        cwReference: commonWell.lastReferenceHeader,
+        cwErrs,
+      },
```

Comment:
Not sure if this is present on the operation outcome, it would be good to have the patient ID here as well.